AutocompleteClient/FW/Logic/Request/CIOQuizzesQuery.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  CIOQuizzesQuery.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+/**
+ Struct encapsulating the necessary and additional parameters required to execute a quizzes query.
+ */
+public struct CIOQuizzesQuery: CIORequestData {
```

Comment:
Let's rename `CIOQuizzesQuery` to `CIOQuizQuery` to be consistent with the other SDKs